corehq/apps/data_analytics/tasks.py

```diff
@@ -0,0 +1,36 @@
+from celery.schedules import crontab
+from celery.task import periodic_task
+from celery.utils.log import get_task_logger
+import datetime
+
+from corehq.apps.data_analytics.malt_generator import MALTTableGenerator
+from dimagi.utils.dates import DateSpan
+from dimagi.utils.django.email import send_HTML_email
+
+
+logger = get_task_logger(__name__)
```

Comment:
The generator catches lot of exceptions and logs them. But I couldn't figure out the log handler that these logs are sent to. I tried using one of [loggers](https://github.com/dimagi/commcare-hq/blob/master/settings.py#L867) defined in settings, but I don't see that being used anywhere else. 
I will put the generator in a try-catch block and send exception using `notify_error`